tests/test_providers.py

```diff
@@ -1,6 +1,6 @@
 import pytest
 import six
-
+import operator
```

Comment:
It would be better to separate the builtin imports from the third party imports, e.g.:

```python
import operator

import pytest
import six
from mock import patch
```